boards/x86/qemu_x86_jailhouse/doc/board.rst

```diff
@@ -0,0 +1,120 @@
+.. _qemu_x86_jailhouse:
+
+Jailhouse cell X86 Emulation (QEMU)
+###################################
+
+Overview
+********
+
+The X86 Jailhouse QEMU board configuration is used to emulate the X86
```

Comment:
your commit message mentions "This is code that brings in configuration to build, on x86, a target that will run as a Jailhouse (https://github.com/siemens/jailhouse) hypervisor guest OS" but there's no mention of this in the document.   Maybe at least add a reference/link to this github site for more information?